lib/ansible/modules/windows/win_package.py

```diff
@@ -243,7 +243,7 @@
 RETURN = r'''
 log:
   description: The contents of the MSI log.
-  returned: change occured and package is an MSI
+  returned: failure during install and package is an MSI
```

Comment:
Maybe write it as `Installation failure for MSI packages`

Reply:
Logs will be present for uninstall as well, so I'll put it as `Installation/uninstallation failure for MSI packages`